app/db/crud/document.py

```diff
@@ -126,20 +126,20 @@ def _get_visible_languages_for_phys_doc(
 
 def get_family_and_documents(db: Session, import_id: str) -> FamilyAndDocumentsResponse:
     """
-    Get a document along with the family information.
+    Get a document along with the corpus information.
 
     :param Session db: connection to db
     :param str import_id: id of document
     :return DocumentWithFamilyResponse: response object
     """
 
     db_objects = (
-        db.query(Family, Geography, FamilyMetadata, FamilyOrganisation, Organisation)
+        db.query(Family, Geography, FamilyMetadata, FamilyCorpus, Corpus)
         .filter(Family.import_id == import_id)
         .join(Geography, Family.geography_id == Geography.id)
         .join(FamilyMetadata, import_id == FamilyMetadata.family_import_id)
-        .join(FamilyOrganisation, import_id == FamilyOrganisation.family_import_id)
-        .filter(FamilyOrganisation.organisation_id == Organisation.id)
+        .join(FamilyCorpus, import_id == FamilyCorpus.family_import_id)
+        .filter(FamilyCorpus.corpus_import_id == Corpus.import_id)
```

Comment:
FYI .. we probably still want to be filtering & getting the organisation too at this point

Reply:
then I am not sure what to change here